src/breadcrumbs/breadcrumbs.ts

```diff
@@ -0,0 +1,18 @@
+import {Component, ContentChildren, Query, QueryList, Input} from 'angular2/core';
+import {NglBreadcrumb} from './breadcrumb';
+
+@Component({
+ selector: 'ngl-breadcrumbs',
+ directives: [NglBreadcrumb],
+ templateUrl: './breadcrumbs.jade',
+})
+export class NglBreadcrumbs {
+  @Input() orientation: 'vertical' | 'horizontal' = 'horizontal';
+  @Input('assistive-text') assistiveText: string = 'You are here:';
```

Comment:
I would prefer not to give a default value as we may later have issues with the i18n.